framework/source/class/qx/util/StringEscape.js

```diff
@@ -37,26 +37,21 @@ qx.Bootstrap.define("qx.util.StringEscape",
     {
       var entity, result = "";
 
-      for (var i=0, l=str.length; i<l; i++)
-      {
-        var chr = str.charAt(i);
-        var code = chr.charCodeAt(0);
+      for (var codePoint of str) {
+        var code = codePoint.codePointAt(0);
 
         if (charCodeToEntities[code]) {
           entity = "&" + charCodeToEntities[code] + ";";
-        }
-        else
-        {
+        } else {
```

Comment:
Uh oh, @derrell will not like this :-)

Reply:
lol. You're right... but this file is already a mess as far as mixed styles. I would prefer that it not be further mixed. In any case, though, formatting changes for the sake of formatting changes should not be included in a PR intending to fix a bug, but rather in its own PR, and in that latter PR, the format should be made consistent throughout the file.

Reply:
I removed any style changes, just concentrating on fixing the issue

Reply:
Thanks.